tests/Testing/Model/Data.php

```diff
@@ -2,7 +2,7 @@
 
 namespace Streamx\Clients\Ingestion\Tests\Testing\Model;
 
-class Page
+class Data
```

Comment:
We will send to Data channel in StreamX Connector, so changed to Data here. May be reverted since Page may be more readable to worldwide users